source/MissionAction.cpp

```diff
@@ -240,9 +243,24 @@ bool MissionAction::CanBeDone(const PlayerInfo &player) const
 		if(flagship)
 			available += flagship->OutfitCount(it.first);
 		
-		// If the gift "count" is 0, that means to check that the player has at
-		// least one of these items.
-		if(available < -it.second + !it.second)
```

Comment:
if you leave `!it.second` in the compare that gets triggered by `outfit ____ 0`, would backwards compatibility for missions already in players' savegames (or other peoples' plugins) be guaranteed?

If so, perhaps restore it and indicate that it remains for that very purpose.